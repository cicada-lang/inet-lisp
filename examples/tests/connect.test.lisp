(define-node sole value!)

(define (main)
  (= x x-op (link))
  (connect x (sole))
  x-op)

(main)
