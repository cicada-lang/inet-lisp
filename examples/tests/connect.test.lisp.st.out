(net
 :root (sole₁)
 :node-count 1
 (sole₁
  :value! ____))
